install.sh

```diff
@@ -2,41 +2,49 @@
 cd `dirname $0`
 set -e
 
+printf "Looking for javac... "
 if [ ! -f /usr/bin/javac ] ; then
-    # Need to update or else the installs won't work
+    printf "... not found. Installing packages... "
+    # Need to update or else the installs won't work. Failing is not critical.
     sudo DEBIAN_FRONTEND=noninteractive apt-get -qq update
 
     # Downgrade tzdata, hilarious.
-    sudo DEBIAN_FRONTEND=noninteractive apt-get -qq install --force-yes -y tzdata/trusty
+    sudo DEBIAN_FRONTEND=noninteractive apt-get -qq install --force-yes -y tzdata/trusty || (echo Failed to install tzdata. ; exit 1)
 
     # Some things we need to build Web-CAT.
-    sudo DEBIAN_FRONTEND=noninteractive apt-get -qq install -y openjdk-7-jre openjdk-7-jdk ant git
+    sudo DEBIAN_FRONTEND=noninteractive apt-get -qq install -y openjdk-7-jre openjdk-7-jdk ant git || (echo Failed to install required packages ; exit 1)
 fi
+echo Done!
 
 fetch()
 {
 	url="$1"
 	file="`basename $url`"
 
-	curl -sSL "$url" -o "$file.tmp" && mv "$file.tmp" "$file"
+	curl -sSL "$url" -o "$file.tmp" && mv "$file.tmp" "$file" || (echo Failed to download $file. ; exit 1)
 }
 
 # install WebObjects
+printf "Installing WebObjects (root access required)... "
 [ -f WOInstaller.jar ] || fetch http://wocommunity.org/tools/WOInstaller.jar
 wodir=/Library/WebObjects/Versions/WebObjects543
 sudo mkdir -p $wodir
-[ -d $wodir/Library/Frameworks/JavaXML.framework ] || (sudo java -jar WOInstaller.jar 5.4.3 $wodir >/dev/null || (sudo rm -rf $wodir && exit 1))
+[ -d $wodir/Library/Frameworks/JavaXML.framework ] || (sudo java -jar WOInstaller.jar 5.4.3 $wodir >/dev/null || (sudo rm -rf $wodir && echo Unable to install WebObjects. && exit 1))
+echo Done!
 
 # install Wonder Frameworks
-[ -f Wonder-Frameworks.tar.gz ] || fetch https://jenkins.wocommunity.org/job/Wonder/lastSuccessfulBuild/artifact/Root/Roots/Wonder-Frameworks.tar.gz
+printf "Installing Wonder Frameworks (root access required)... "
+[ -f Wonder-Frameworks.tar.gz ] || fetch https://github.com/wocommunity/wonder/releases/download/wonder-6.1.4/Wonder-Frameworks.tar.gz
 sudo tar xfz Wonder-Frameworks.tar.gz -C $wodir/Library/Frameworks
+echo Done!
 
-# woproject.jar
-[ -f woproject.jar ] || fetch http://webobjects.mdimension.com/hudson/job/WOLips36Stable/lastSuccessfulBuild/artifact/woproject.jar
+# woproject.jar is in the repository.
```

Comment:
FWIW, it's a little sad that we embed woproject.jar, but I'm just commenting that, not saying that you need to change what you're doing.